docs/source/guides/dfs_with_dask_entitysets.rst

```diff
@@ -0,0 +1,121 @@
+
+
+Now that we have our Dask dataframe, we can start to create the ``EntitySet``. The current implementation does not support variable type inference for Dask entities, so we must pass a dictionary of variable types using the ``variable_types`` parameter when calling ``es.entity_from_dataframe()``. Aside from needing to supply the variable types, the rest of the process of creating an ``EntitySet`` is the same as if we were using pandas dataframes.
+
+.. ipython:: python
+
+    es = ft.EntitySet(id="dask_es")
+    es = es.entity_from_dataframe(entity_id="dask_entity",
+                                  dataframe=dask_df,
+                                  index="id",
+                                  variable_types={"id": ft.variable_types.Id,
+                                                  "values": ft.variable_types.Numeric})
+    es
+
+
+Notice that when we print our ``EntitySet``, the number of rows for the ``dask_entity`` entity is returned as a Dask ``Delayed`` object. This is because obtaining the length of a Dask dataframe requires an expensive compute operation to sum up the lengths of all the individual partitions that make up the dataframe and that operation is not performed by default.
+
+
+Running DFS
+-----------
+We can pass the ``EntitySet`` we created above to ``featuretools.dfs`` in order to create a feature matrix. If the ``EntitySet`` we pass to ``dfs`` is made of Dask entities, the feature matrix we get back will be a Dask dataframe.
+
+.. ipython:: python
+
+    feature_matrix, features = ft.dfs(entityset=es,
+                                      target_entity="dask_entity",
+                                      trans_primitives=["negate"])
+    feature_matrix
+
+
+This feature matrix can be saved to disk or computed and brought into memory, using the appropriate Dask dataframe methods.
+
+.. ipython:: python
+
+    fm_computed = feature_matrix.compute()
+    fm_computed
+
+
+While this is a simple example to illustrate the process of using Dask dataframes with Featuretools, this process will also work with an ``EntitySet`` containing multiple entities, as well as with aggregation primitives.
+
+Limitations
+-----------
+There are many parts of Featuretools that are difficult to implement in a distributed environment and several primitives that are not well suited to operate on distributed dataframes. As a result, there are some limitations when creating a Dask ``Entityset`` and then using it to generate a feature matrix. The most significant limitations are reviewed in more detail in this section.
+
+Supported Primitives
+********************
+When creating a feature matrix from a Dask ``EntitySet``, only certain primitives can be used. Primitives that rely on the order of the entire dataframe or require an entire column to be computed are inefficient and difficult to support in a distributed environment. As a result, only a subset of Featuretools primitives are currently supported when using a Dask ``EntitySet``.
```

Comment:
can we indicate that some of these limitations are no because it is `inefficient and difficult to support`, but that because we just haven't done it yet? and that users should file issues on github to request missing functionality.

maybe this comment can actually go in the note up top

Reply:
also, can we try to phrase this that the key functionality is there, but we are documenting the long tail of missing stuff for completeness. I dont want to scare users away with the limitations

Reply:
Updated the wording in the limitations section as well as the note to improve based on these suggestions.